doc/tutorials/Content_Page.md

```diff
@@ -0,0 +1,86 @@
+# Content Page
```

Comment:
Maybe add `setup` to the name of the `docker` folder for consistency with the rest?